modules/programs/newsboat.nix

```
@@ -102,7 +102,7 @@ in {
 
       mkQueryEntry = n: v: ''"query:${n}:${escape [ ''"'' ] v}"'';
       queries = mapAttrsToList mkQueryEntry cfg.queries;
-    in concatStringsSep "\n" (urls ++ queries) + "\n";
+    in concatStringsSep "\n" (queries ++ urls) + "\n";
```

How to use state version:
```suggestion
    in concatStringsSep "\n" (
    if versionAtLeast config.home.stateVersion "20.03"
    then queries ++ urls
    else urls ++ queries
    ) + "\n";
```

Note, you'll need to add an entry to the "State Version Changes" section of the `rl-2003.adoc` file describing the change. It may be worth having two tests, the old one + one with `home.stateVersion = 20.03`.

---

Great! I'll do just that.

---

The old one will fail in the future, right? When the testing context is newer than 19.09 I mean.